downstream/modules/hub/proc-create-synclist.adoc

```diff
@@ -34,15 +35,9 @@ The following domain names must be in the allow list:
 
 .Procedure
 
-. Log in to `{Console}`.
-. Navigate to menu:Automation Hub[Collections].
-. Set the *Sync* toggle switch on each collection to exclude or include it on your synclist.
-+
-[NOTE]
-====
-You will only see the *Sync* toggle switch if you have administrator permissions.
-====
-+
-. To initiate the remote repository synchronization, navigate to your {PlatformNameShort} and select {MenuACAdminRepositories}.
-. In the row containing the repository you want to sync, click the btn:[More Actions] icon *{MoreActionsIcon}* and select *Sync repository* to initiate the remote repository synchronization to your {PrivateHubName}. 
-. Optional: If your remote repository is already configured, update the collections content that you made available to local users by manually synchronizing Red Hat {CertifiedName} to your {PrivateHubName}.
+. From the navigation panel, select {MenuACAdminRemotes}.
+. Find the remote you want to sync from and click the pencil icon to edit. 
+. Find the field labeled *Requirements file*. There, you can either paste the contents of your requirements file, or upload the file from your hard drive by selecting the upload button.
+. Click btn:[Save remote].  
+. To begin synchronization, from the navigation panel select {MenuACAdminRepositories}.
+. In the row containing the repository you want to sync, click the btn:[More Actions] icon *{MoreActionsIcon}* and select *Sync repository* to initiate the remote repository synchronization to your {PrivateHubName}. 
```

Comment:
There's no btn:[More Actions] the attribute contains that I think. Just click the {MoreActionsIcon} icon and select the image:sync.png[Sync repository] *Sync Repository* icon.